u-boot/cpu/mips/ar7240/qca_dram.c

```diff
@@ -624,6 +629,11 @@ static inline void qca_dram_set_ddr_cfg2(u32 mem_cas,
 
 	/* Gate open latency = 2 * MEM_CAS */
 	tmp = 2 * mem_cas;
+#if (SOC_TYPE & QCA_AR933X_SOC)
```

Comment:
I just ran a quick test before I left the office, it crashed when I don't set DDR_CONFIG2. 

Unfortunately, it looks like both have to be set (aligned), given the spec says that CAS_LATENCY has to be >= GATE_OPEN_LATENCY there is no point trying DDR_CONFIG2 only.

I'll double check tomorrow, and let you know.